src/commands/createContainerApp/ContainerAppCreateStep.ts

```diff
@@ -63,6 +63,9 @@ export class ContainerAppCreateStep extends AzureWizardExecuteStep<IContainerApp
         progress.report({ message: creatingSwa });
         ext.outputChannel.appendLog(creatingSwa);
 
+        context.image ||= `${getLoginServer(context)}/${context.repositoryName}:${context.tag}`;
+        const name = context.image.substring(context.image.lastIndexOf('/') + 1).replace(':', '-');
```

Comment:
Just to double check the logic here--some full image names will have several slashes even after the registry, e.g.:

`mcr.microsoft.com/dotnet/aspnet:5.0`

In that, the registry is `mcr.microsoft.com`, the repository is `dotnet/aspnet`, and the tag `5.0`. The name chosen would be `aspnet-5.0`, instead of `dotnet/aspnet-5.0`.

I'm not saying this is wrong, just want to make sure this is the behavior you are aiming for.

Reply:
Yeah, the name doesn't really matter, it just has to be unique to that container so just wanted to grab something + the tag to make it unique.